test/active_model_serializers/model_test.rb

```diff
@@ -18,5 +27,59 @@ def test_initialization_with_string_keys
 
       assert_equal model_instance.read_attribute_for_serialization(:key), value
     end
+
+    def test_attributes_can_be_read_for_serialization
+      klass = subclass_without_caching_support do
+        attributes :one, :two, :three
+      end
+      original_attributes = { one: 1, two: 2, three: 3 }
+      instance = klass.new(original_attributes)
+
+      # Initial value
+      expected_attributes = { one: 1, two: 2, three: 3 }.with_indifferent_access
+      assert_equal expected_attributes, instance.attributes
+      assert_equal 1, instance.one
+      assert_equal 1, instance.read_attribute_for_serialization(:one)
+
+      # Change via accessor
+      instance.one = :not_one
+
+      expected_attributes = { one: :not_one, two: 2, three: 3 }.with_indifferent_access
+      assert_equal expected_attributes, instance.attributes
+      assert_equal :not_one, instance.one
+      assert_equal :not_one, instance.read_attribute_for_serialization(:one)
+    end
+
+    def test_id_attribute_can_be_read_for_serialization
```

Comment:
no longer necessary now that the `read_attribute_for_serialization` method has been removed